paasta_tools/cli/cmds/logs.py

```diff
@@ -328,16 +372,37 @@ def get_log_reader():
 
 
 class LogReader(object):
-    def __init__(self, **kwargs):
-        pass
+    def __init__(self):
+        # Tailing, i.e actively viewing logs as they come in
+        self.SUPPORTS_TAILING = False
```

Comment:
I think what @Rob-Johnson is saying is that if you want to treat it like a constant, move it out of the constructor into the top level scope. If you are going to have it in the constructor where it will be set for each instance of the class, rename it to be in lowercase.